crates/binjs_io/src/context/prelude.rs

```diff
@@ -31,10 +46,19 @@ impl Compression {
             _ => None,
         }
     }
+
+    /// Return a file extension for this compression format.
+    /// Used typically for dumping samples.
+    pub fn extension(&self) -> &'static str {
+        match *self {
+            Compression::Brotli => "bro",
```

Comment:
"br" maybe?
https://github.com/google/ngx_brotli#brotli_static

Reply:
Works for me.

I seem to remember used to produce `.bro` files, then the word "bro" became associated to sexual harassment (I think) in the US and they renamed it to `.br`.

Anyway, it's a simple enough patch :)